src/pages/ServiceDetails/ServiceInfo.tsx

```diff
@@ -113,19 +109,19 @@ class ServiceInfo extends React.Component<ServiceDetails, ServiceInfoState> {
     const destinationRules = this.props.serviceDetails.destinationRules || [];
     const validations = this.props.validations || {};
     const validationChecks = this.validationChecks();
-    const getSeverityIcon: any = (severity: ValidationTypes = ValidationTypes.Error) => (
+    const getSeverityIcon: any = (severity: string = 'error') => (
       <span className={tabIconStyle}>
         {' '}
-        <Validation severity={severity} />
+        <Icon type="pf" name={severityToIconName(severity)} />
```

Comment:
Oopsie! Something wrong here. This part is reverting the validation migrations into PF3.

Reply:
I love github rebases :( wtf happen here ...I'll fix this